src/parser.h

```diff
@@ -18,7 +18,7 @@ Parameter parser(int argc, char *argv[])
 {
     Parameter out_param;
 
-    for(std::size_t idx=0; idx<argc; ++idx)
+    for(auto idx=0; idx<argc; ++idx)
```

Comment:
A warning was referring to a type mismatch between `argc` that is an `int` value, and `std::size_t` that is an `unsigned int` value.